crates/snapbox/src/data.rs

```diff
@@ -110,10 +107,10 @@ impl Data {
     /// Return the underlying `str`
     ///
     /// Note: this will not inspect binary data for being a valid `str`.
-    pub fn as_str(&self) -> Option<&str> {
+    pub fn render(&self) -> Option<String> {
```

Comment:
This is unrelated to the rest of the commit.  Could you split this into its own commit?